appserver/admingui/devtests/src/test/java/org/glassfish/admingui/devtests/BaseSeleniumTestClass.java

```diff
@@ -490,8 +506,11 @@ protected void clickLink(String linkText) {
     }
 
     protected void sleep(int millis) {
+        final Object LOCK = new Object();
```

Comment:
Not sure why you had to change this?

Reply:
We faced some issue waiting for one thread longer time. So, tried this one. As tests are stable, i will try without this step and update here.